config/initializers/git.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+raise "Git not found" unless version = `git --version`[/\d+\.\d+\.\d+/]
+raise "Need git v2.0.0+" if Gem::Version.new(version) < Gem::Version.new("2.0.0")
```

Comment:
I think worktree was added in git 2.5.0, if I searched correctly. See: https://github.com/git/git/blob/master/Documentation/RelNotes/2.5.0.txt#L45-L47

Also, see GitHub posts: https://blog.github.com/2015-07-29-git-2-5-including-multiple-worktrees-and-triangular-workflows/ The footnote says that the contrib script was improved to become the worktree subcommand.

But maybe I'm missing something?

Although, the changeling says it's experimental and it might change. Maybe it's safer to check for newer versions, as not all subcommands used might have been implemented by 2.5.0. But might probably be correct.

I can try to test it with git 2.5.0, if you want (and I find a git 2.5.0 build easy to install :D)